src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
```

Comment:
[Not necessary] This file could get some Ramda-love :D

Reply:
Understand, will have a look 🙂

Reply:
There are also some other candidates like assigning values in hadlers in `Filters.tsx` but at least you don't mutate original values outside those methods. But I assume we can Ramdify later on

Reply:
I've covered a few functions by Ramda here, `Filters.tsx` will leave for next PR